nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-avro-record-utils/src/test/resources/org/apache/nifi/avro/defaultArrayInRecords1.json

```diff
@@ -16,7 +16,7 @@
                         "items":"int"
                      },
                      "doc":"array of ints",
-                     "default": 0
+                     "default": [ 0 ]
```

Comment:
Not sure why the first one ever worked because the `0` literal is not an array. It broke with the update to 1.11